applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -9,10 +9,21 @@
 # Import applications
 import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
 
+try:
```

Comment:
we should NOT create a dependency of the structural_mechanics_solver on the meshingapp

why is this needed btw?